### PR TITLE
feat(account): Add labels to the account object

### PR DESCRIPTION
**Pull request by jfagoagas**
#### Related Issue: 

Implements #1025 

#### Description of changes:
- Add `labels` to the `Account` object.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -54,6 +54,7 @@ Thankyou! -->
     5. Added `List`, `Encrypt` and `Decrypt` activities to `datastore` event class. #989 
     6. Added `file` attribute to `http`, `rdp`, `ssh`, and `ftp` event classes. #985
     7. Added a `Preauth` `activity_id` to the `Authentication` class. #1018
+    8. Added a `labels` to the `Account` object.
```
Comment (floydtree) [inline CHANGELOG.md:57]: This will need to be added to the Improved->Objects section.

**Review by floydtree — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (jfagoagas) [reply CHANGELOG.md:57]: You are right, changing it.

**Review by floydtree — APPROVED** [review]
Thanks for getting this in!

**Review by pagbabian-splunk — APPROVED** [review]

**Review by zschmerber — APPROVED** [review]
Seems useful thanks for the add

**Pull request merged**